backend-api/src/functions/asyncCredential/asyncCredentialHandler.test.ts

```diff
@@ -226,343 +231,37 @@ describe("Async Credential", () => {
-          });
-
-          expect(result).toStrictEqual({
-            headers: { "Content-Type": "application/json" },
-            statusCode: 400,
-            body: JSON.stringify({
-              error: "invalid_token",
-              error_description:
-                "iss claim does not match ISSUER environment variable",
-            }),
-          });
-        });
-      });
-    });
-
-    describe("scope claim validation", () => {
-      describe("Given scope is missing", () => {
-        it("Returns a log", async () => {
-          const jwtBuilder = new MockJWTBuilder();
-          jwtBuilder.deleteScope();
-          const event = buildRequest({
-            headers: { Authorization: `Bearer ${jwtBuilder.getEncodedJwt()}` },
-          });
-
-          dependencies.tokenService = () => new MockTokenSeviceValidSignature();
-
-          const result: APIGatewayProxyResult = await lambdaHandler(
-            event,
-            dependencies,
-          );
-          expect(mockLogger.getLogMessages()[0].logMessage.message).toBe(
-            "JWT_CLAIM_INVALID",
-          );
-          expect(mockLogger.getLogMessages()[0].data).toStrictEqual({
-            errorMessage: "Missing scope claim",
-          });
-
-          expect(result).toStrictEqual({
-            headers: { "Content-Type": "application/json" },
-            statusCode: 400,
-            body: JSON.stringify({
-              error: "invalid_token",
-              error_description: "Missing scope claim",
-            }),
-          });
-        });
-      });
-
-      describe("Given scope is not dcmaw.session.async_create", () => {
-        it("Returns a log", async () => {
-          const jwtBuilder = new MockJWTBuilder();
-          jwtBuilder.setScope("invalidScope");
-          const event = buildRequest({
-            headers: { Authorization: `Bearer ${jwtBuilder.getEncodedJwt()}` },
-          });
-
-          dependencies.tokenService = () => new MockTokenSeviceValidSignature();
-
-          const result: APIGatewayProxyResult = await lambdaHandler(
-            event,
-            dependencies,
-          );
-          expect(mockLogger.getLogMessages()[0].logMessage.message).toBe(
-            "JWT_CLAIM_INVALID",
-          );
-          expect(mockLogger.getLogMessages()[0].data).toStrictEqual({
-            errorMessage: "Invalid scope claim",
-          });
-
-          expect(result).toStrictEqual({
-            headers: { "Content-Type": "application/json" },
-            statusCode: 400,
-            body: JSON.stringify({
-              error: "invalid_token",
-              error_description: "Invalid scope claim",
-            }),
-          });
+  describe("Get decoded token", () => {
+    describe("Given decoding token fails", () => {
+      it("Logs and returns 400 Bad Request response", async () => {
+        const jwtBuilder = new MockJWTBuilder();
+        jwtBuilder.deleteExp();
```

Comment:
If we're controlling the mock, do we need to delete this here?

Reply:
Done